src/legacy/core_plugins/kibana/public/discover/components/field_chooser/discover_index_pattern.test.tsx

```diff
@@ -48,28 +48,4 @@ describe('DiscoverIndexPattern', () => {
 
     expect(shallowWithIntl(<DiscoverIndexPattern {...props} />)).toMatchSnapshot(`""`);
   });
-  test('A single index pattern is just displayed', () => {
```

Comment:
The behaviour is different for lens so this test can be removed unless it should be corrected in lens too? 
![image](https://user-images.githubusercontent.com/4283304/69975643-db936800-1527-11ea-86aa-296cc77fec95.png)

Reply:
I think it's fine to remove it